### PR TITLE
Feat/auth/signup

### PR DESCRIPTION
**Pull request by abhij1607**
This PR consist of-

- added signup page link in login page
- signup page and functionality

### CODE REVIEW

**Review by stutisk — APPROVED** [review]
liked the logic implemented 
good work with UI 
can go forward with merging it

File: src/utils/server-request.js [hunk 1]
```diff
@@ -28,4 +28,15 @@ const requestLogin = async (dispatch, body, navigate, location) => {
   }
 };
 
-export { fetchVideos, fetchCategories, requestLogin };
+const requestSignUp = async (dispatch, requestBody, navigate, location) => {
+  try {
+    const response = await axios.post("/api/auth/signup", requestBody);
+    console.log(response);
```
Comment (Tanay-J) [inline src/utils/server-request.js]: You may remove this

**Review by Tanay-J — APPROVED** [review]
Code is quite readable complete with good coding practices. 
You may merge.

File: src/utils/server-request.js (hunk 1 above)
Comment (abhij1607) [reply src/utils/server-request.js]: Thanks for pointing this out

**Pull request merged**